### DISCUSSION TITLE
Links stopped working in markup

### DISCUSSION THREAD

**Discussion by Konstantinos-Chatzistavrou** [community/community#102608] [Mobile]
### Select Topic Area

Bug

### Body

So, I have a table of contents in my repository's README.md file and it was working fine, I was clicking the links and it scrolled to the right category but I noticed that for the past week this isn't working. I tried editing, changing some things adding characters but it just doesn't work, the table of contents just lost its functionality. Please fix it.

**Comment by ghost** [comment]
can you provide the link for the Repository so we could help

**Reply by Konstantinos-Chatzistavrou** [reply]
Sure here is the Link: https://github.com/kochas23/Threat-Watch
